README.rst

```diff
@@ -1,23 +1,24 @@
+######################
 Debug printing library
-======================
+######################
 
 Overview
---------
+========
```

Comment:
Ideally the title scheme recommended in python docs

Reply:
It seems to cause problems with the section nesting though, because the changelog is included at the end and the changelog has = underlining at it's highest level, so it can be embedded into another section.